packages/svelte-ux/src/routes/docs/utils/format/+page.svelte

```diff
@@ -83,13 +102,277 @@
   <div>{format(0.5678, 'percent', { locales: 'fr', fractionDigits: 1 })}</div>
 </Preview>
 
-<h2>Period formats</h2>
+<h1>Dates</h1>
+
+<h2>No format</h2>
+
+<Preview>
+  {format(myDate)}
+  <!-- TODO JYC?
+    before: 30/03/1982 00:00:00
+    after: Tue Mar 30 1982 00:00:00 GMT+0200 (heure d’été d’Europe centrale)
+  -->
+</Preview>
+
+<h2>Custom format</h2>
+
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>With random string</h3>
+    <Preview>
+      {format(myDate, PeriodType.Custom, {
+        custom: 'eee, MMMM do',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>With descriptive tokens</h3>
+    <Preview>
+      {format(myDate, PeriodType.Custom, {
+        custom: [DateToken.DayOfWeek_short, DateToken.Month_long, DateToken.DayOfMonth_withOrdinal],
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>With full intl</h3>
+    <Preview>
+      {format(myDate, PeriodType.Custom, {
+        custom: { weekday: 'short', month: 'long', day: 'numeric', withOrdinal: true },
+      })}
+    </Preview>
+  </div>
+</div>
+
+<h2>PeriodType Day</h2>
+
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>short</h3>
+    <Preview>
+      {format(myDate, PeriodType.Day, {
+        variant: 'short',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>default</h3>
+    <!-- TODO JYC QUESTION: before: 3/30/1982, suggestion as default: 03/30/1982 -->
```

Comment:
It's back to what exist today in prod, I wonder if it's not better with leading 0 (even in `en`)?
I let you tell me ^^

Reply:
I'd prefer non-leading 0s by default.  It's also the default for `new Intl.DateTimeFormat("en-US").format(...)`

![image](https://github.com/techniq/svelte-ux/assets/177476/3962a53a-f4b9-4f60-a042-85940b63f7a5)

Reply:
I won't say there isn't a case for leading 0s... lining up dates in a table column is so much nicer with leading 0s, but also I think it's a little easier to read without, and when you are space constrained, it can be nice to not have 0s as well.  At least it's configurable :)

Reply:
`new Intl.DateTimeFormat("fr").format(new Date())` has leading 0 ;)

It's all configurable, so it's OK. but find it "strange" to have a default parsing different than a default display.
I'll remove the comment and resolve here 👍 (until it bugs me more ^^)